README.md

```diff
@@ -254,6 +295,7 @@ If you want a similar solution for your iOS projects, check this out: [poeditor-
 ## Authors & Collaborators
 * **[Iván Martínez](https://github.com/imartinez)** - *Initial work*
 * **[Adrián García](https://github.com/adriangl)** - *Maintainer*
+* **[sonnet](https://github.com/rafid059)** - *Contributor*
```

Comment:
@rafid059 Is this OK for you? I usually mention all people that contributed code to the project here